docs/schema-generator/customizing-schemas/renaming-fields.md

```diff
@@ -42,7 +58,14 @@ input MyInputClassInput {
   renamedField: String!
 }
 
+# You should keep UPPERCASE naming if possible in GraphQL enums, but any case is supported
```

Comment:
this comment is not defined as part of the schema above -> should we remove it to avoid confusion? (or move it to the NOTE section above_

Reply:
Yea, we can move it above into the code comment on enums specifically